kong/plugins/datadog/schema.lua

```diff
@@ -0,0 +1,8 @@
+return {
+  fields = {
+    host = { type = "string", default = "localhost" },
```

Comment:
It's fine for now but in the future make sure you use the same style as most of the codebase is using:

```
{key = value}
not
{ key = value }
```

Parts using the style with spaces are simply not following the trend over the codebase. I am always finding myself fixing those all around the repo, because the spaces style keeps being used when it should not.